src/helm/config/tokenizer_configs.yaml

```diff
@@ -211,6 +211,20 @@ tokenizer_configs:
     end_of_text_token: "</s>"
     prefix_token: "<s>"
 
+
+  # Allen Institute for AI
+  # The allenai/olmo-7b requires Python 3.9 or newer.
+  # To use the allenai/olmo-7b tokenizer, run `pip install ai2-olmo` first.
```

Comment:
Or should it be `pip install crfm-helm[allenai]` to encapsulate the dependencies?

Reply:
Sure.